contributors/devel/conformance-tests.md

```diff
@@ -1,57 +1,69 @@
 # Conformance Testing in Kubernetes
 
-The Kubernetes conformance test suite is a set of testcases, currently a
-subset of the integration/e2e tests, that the Architecture SIG has approved
-to define the core set of interoperable features that all Kubernetes
-deployments must support.
+The Kubernetes Conformance test suite is a subset of e2e tests that 
+SIG Architecture has approved to define the core set of interoperable 
+features that all Kubernetes clusters must support. The tests verify
```

Comment:
I wouldn't use the word "must".  Folks could fail tests and not get the badging.

Reply:
Do we have examples of tests that are allowed to fail but people can still claim conformance?

Reply:
I don't understand, are you saying folks should be allowed to fail conformance tests but get certified as conformant anyway?

Reply:
I'd suggest changing "all Kubernetes clusters must support" to "all conformant Kubernetes clusters must support".